.github/workflows/rust.yml

```diff
@@ -122,4 +122,9 @@ jobs:
           command:                 check
           toolchain:               stable
           args:                    --manifest-path ./frame-metadata/Cargo.toml --no-default-features --features v12,v13,v14
-
+      - name:                      Checking fully featured wasm
+        uses:                      actions-rs/cargo@v1
+        with:
+          command:                 check
+          toolchain:               stable
+          args:                    --manifest-path ./frame-metadata/Cargo.toml --no-default-features --features decode,serde_full,v14
```

Comment:
Maybe also worth adding a specific check for just `serde_full` and just `decode` (it's so easy to miss compile errors that don't occur when both are enabled but do occur when just one is)

Reply:
I suggest that we try to use `cargo hack check --each-feature` here if it's possible instead doing this manually it is so easy to some combination of feature flags.